<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>191</width>
      <height>361</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>значения</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_avg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>max min avg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gnom">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Гномья</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_monkey">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обезьянья</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rashectka">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Расческа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quick">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Быстрая</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bubble">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пузырьковая</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_avg">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_found">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_searchCount">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteDublikat">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>480</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удаление дубликатов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_all_clear">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Очищаем всё!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rundom">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>281</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string>Рандом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>поиск</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
